lib/modules/dosomething/dosomething_helpers/dosomething_helpers.share.inc

```diff
@@ -26,7 +26,7 @@ function dosomething_helpers_share_redirect() {
  *
  * @return string   The facebook intent url.
  */
-function dosomething_helpers_get_facebook_intent($link, $type = NULL, $custom_options = NULL) {
+function dosomething_helpers_get_facebook_intent($type = NULL, $link, $custom_options = NULL) {
```

Comment:
@angaither ah ok. I thought this would be more readable but that makes more sense.